### PR TITLE
Fix builds

### PR DESCRIPTION
**Pull request by IllusionMan1212**
Turns out the builds errors because i only searched for PRs with the latest commit and didn't take into consideration commits done directly to the repo

I also added the new links for linux and mac

### CODE REVIEW

File: Vita3KBot/APIClients/AppveyorClient/AppVeyorClient.cs [hunk 1]
```diff
@@ -33,22 +33,30 @@ public static class AppveyorClient {
...
+            if (prInfo != null) {
+                LatestBuild.WithTitle($"PR: #{prInfo.Number} By {JobJSON.Build.AuthorUserName}")
+                .WithUrl(prInfo.HtmlUrl);
+            } else {
+                LatestBuild.WithTitle($"Commit: {JobJSON.Build.CommitId} By {JobJSON.Build.AuthorUserName}")
+                .WithUrl($"https://github.com/vita3k/vita3k/commit/{JobJSON.Build.CommitId}");
+            }
+
+            LatestBuild.WithDescription($"{JobJSON.Build.Message}")
+            .WithColor(Color.Orange)
```
Comment (1whatleytay) [inline Vita3KBot/APIClients/AppveyorClient/AppVeyorClient.cs]: I usually indent this stuff...

File: Vita3KBot/APIClients/AppveyorClient/AppVeyorClient.cs [hunk 2]
```diff
@@ -33,22 +33,30 @@ public static class AppveyorClient {
...
+            .AddField("Windows", $"[{FileName}](https://ci.appveyor.com/api/buildjobs/{JobId}/artifacts/{FileName})")
+            .AddField("Linux", $"[{latestRelease.Assets[0].Name}]({latestRelease.Assets[0].BrowserDownloadUrl})")
+            .AddField("Mac", $"[{latestRelease.Assets[1].Name}]({latestRelease.Assets[1].BrowserDownloadUrl})");
 
             return LatestBuild.Build();
         }
 
-        public static async Task<Issue> GetPRInfo(Build build) {
-            var github = new GitHubClient(new ProductHeaderValue("Vita3KBot"));
+        public static async Task<Issue> GetPRInfo(GitHubClient github, Build build) {
```
Comment (1whatleytay) [inline Vita3KBot/APIClients/AppveyorClient/AppVeyorClient.cs]: Extra space here is a bit odd.

File: Vita3KBot/APIClients/AppveyorClient/AppVeyorClient.cs (hunk 1 above)
Comment (IllusionMan1212) [reply Vita3KBot/APIClients/AppveyorClient/AppVeyorClient.cs]: Noted!, i keep forgetting to conform to the style

File: Vita3KBot/APIClients/AppveyorClient/AppVeyorClient.cs (hunk 2 above)
Comment (IllusionMan1212) [reply Vita3KBot/APIClients/AppveyorClient/AppVeyorClient.cs]: Weird space indeed lol, i must've missed it. will fix it in a later PR

**Pull request merged**